src/js/background.js

```diff
@@ -817,4 +821,4 @@ startBackgroundListeners();
 
 // TODO move listeners and this message behind INITIALIZED
 console.log('Privacy badger is ready to rock!');
-console.log('Set DEBUG=1 to view console messages.');
+console.log('Set DEBUG=1 to view console messages.');
```

Comment:
Style nit: missing end-of-file newline. This change shouldn't be here, basically.

Reply:
Hey Alexei,

Would you prefer that I remove this change from the original commit OR make a new commit to remove the change? (Kai's right beside me telling me my editor added the change without my knowledge).

Reply:
Probably more straightforward to do followup commits (rather than rewrite history).

(I'm about to step out, won't be able to respond quickly, sorry.)